### DISCUSSION TITLE
paused dag continues to execute

### DISCUSSION THREAD

**Discussion by iRecursion** [apache/airflow#27090] [General]
# Envirionment
airflow 2.4.1，python3.9，ubuntu 20.04 lts

# csv.py
```python
import pandas as pd
from datetime import datetime

sample_data = pd.DataFrame([[1,2],[2,3]])

pattern = '%Y%m%d %H:%M:%S %f'
csv_path = r'/opt/airflow/dags/csv/{}.csv'.format(datetime.now().strftime('%Y.%m.%d %H:%M:%S'))

sample_data.to_csv(csv_path)
```

# airflow_repeat_when_dag_paused.py
```python
from airflow import DAG
from airflow.operators.bash import BashOperator
from datetime import datetime

with DAG(
    dag_id="paused_dag_repeat_work_dag",
    start_date=datetime(2022,10,17),
    schedule_interval='*/30 * * * * *',
    catchup=False) as dag:

    task_shell = BashOperator(
        task_id="task_shell",
        bash_command="python /opt/airflow/dags/csv.py"
    )
    task_shell
```

# AirFlow WebUI
[![](https://github.com/iRecursion/repo-discussion/blob/main/Image/Discussion/01PausedDag.png?raw=true)]()

[![](https://github.com/iRecursion/repo-discussion/blob/main/Image/Discussion/02DaemonWorking.png?raw=true)]()

**Comment by iRecursion** [comment]
even if using `docker compose down` to cleanup，
when execute `docker compose --env-file .env up -d`，
the problem remains.

**Comment by iRecursion** [comment]
https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#config-scheduler-min-file-process-interval

**Comment by iRecursion** [comment]
refresh time is not the schedule_interval?
is it possible that it is a radio box in WebUI(Paused)？

**Comment by potiuk** [comment]
I have no idea what you are asking for. I suggest - if you really want help - to precisely describe your problem, expectations you have etc. Those points you made there are loosely disconnected sentences that make no sense as problem statement.
